server/index.js

```diff
@@ -33,41 +33,72 @@ app.use((req, res, next) => {
   next()
 })
 
-// Remove trailing slashes
+// API Docs - resolve before trailing slash redirect so assets don't break
+
+app.use(
+  '/developers/api',
+  express.static(path.join(__dirname, '../api-docs/build/'), {
+    redirect: false,
+  })
+)
+
+// Remove trailing slashes, UNLESS
+// A) We're in local dev mode, in which case we need the slashes to communicate with the Jekyll server
+// B) Request is for an API docs landing, which needs the slash to not break assets
+// If A or B is true, ensure trailing slash is there
+
+const hasTrailingSlash = reqPath => /.+\/$/.test(reqPath)
+const isApiDocs = reqPath => /^\/developers\/api\/?$/.test(reqPath)
 app.get('*', (req, res, next) => {
-  if (/.+\/$/.test(req.path)) {
+  if (
+    hasTrailingSlash(req.path) &&
+    !(process.env.LOCAL || isApiDocs(req.path))
```

Comment:
:bulb: Might we be able to consolidate on `NODE_ENV` for this? Might be nice to save ourselves the extra env var.

Reply:
I think that's a good call. Will update!